### PR TITLE
Printing tests as part of a Zuul source output

### PR DESCRIPTION
**Pull request by jsanemet**
Adding test cases as part of the output of a Zuul source for a '--tests' query.

### CODE REVIEW

File: cibyl/outputs/cli/ci/system/impls/zuul/colored.py [hunk 1]
```diff
@@ -213,8 +215,95 @@ def print_build(self, build: Build) -> str:
                 if build.duration.value:
                     result.add(get_duration_section(self.palette, build), 1)
 
+            if self.query >= QueryType.TESTS:
+                result.add(self.palette.blue('Test Suites: '), 1)
```
Comment (cescgina) [inline cibyl/outputs/cli/ci/system/impls/zuul/colored.py:219]: Shouldn't we add a guard here to catch the cases of jobs that have no builds? Like we do with pipelines and projects. Also, in those cases I would add a message like `No tests found in query`

Comment (jsanemet) [reply cibyl/outputs/cli/ci/system/impls/zuul/colored.py:219]: You are right, I am missing that.

Comment (jsanemet) [reply cibyl/outputs/cli/ci/system/impls/zuul/colored.py:219]: Done.

**Review by cescgina — APPROVED** [review]
LGTM

**Pull request merged**